Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransporter.cs

```diff
@@ -260,6 +294,11 @@ private static ConnectionInfo GetConnectionInfo(Destination destination, Connect
         return connectionInfo;
     }
 
+    private static void CheckServerFingerprint(SftpClient client)
```

Comment:
?

Reply:
This is now used as the method to check the server fingerprint